solidity-v1/dashboard/src/services/threshold-authorization.service.js

```diff
@@ -0,0 +1,101 @@
+import {
+  ContractsLoaded,
+  getContractDeploymentBlockNumber,
+  getThresholdTokenStakingAddress,
+} from "../contracts"
+import { getAllOperatorStakedEventsByAuthorizer } from "./token-staking.service"
+import {
+  AUTH_CONTRACTS_LABEL,
+  TOKEN_GRANT_CONTRACT_NAME,
+} from "../constants/constants"
+import { Keep } from "../contracts"
+import { isSameEthAddress } from "../utils/general.utils"
+import { gt } from "../utils/arithmetics.utils"
+
+const fetchThresholdAuthorizationData = async (address) => {
+  if (!address) {
+    return []
+  }
+  const thresholdTokenStakingContractAddress = getThresholdTokenStakingAddress()
+  const { stakingContract, grantContract } = await ContractsLoaded
+  const keepOperatorStakedEvents = await getAllOperatorStakedEventsByAuthorizer(
+    address
+  )
+  const authorizerOperators = keepOperatorStakedEvents.map(
+    (_) => _.returnValues.operator
+  )
+  const authorizationData = []
+
+  const keepToTStakedEvents =
+    await Keep.keepToTStaking.getStakedEventsByOperator(authorizerOperators)
+
+  const operatorsStakedToT = keepToTStakedEvents.map(
+    (event) => event.returnValues.stakingProvider
+  )
```

Comment:
What do you think about convert `operratorrsStakedTo` to map or object? It will be easier to set correct values for `isStakedToT` and `isFromGrant` flags.
map
```
const operatorsStakedToT = new Map(keepToTStakedEvents.map(_ => [_.returnValues.stakingProvider, {... _.returnValues}]));
```

object
```
const operatorsStakedToT = keepToTStakedEvents.reduce((map, _) => {
    map[_.returnValues.stakingProvider] = {... _.returnValues }
    return map
}, {})
```